companies/1blu.json

```diff
@@ -18,6 +18,8 @@
         "https://www.1blu.de/datenschutz/",
         "https://www.1blu.de/impressum/"
     ],
+    "request-language": "it",
+    "custom-access-template": "access-evangelische-kirche",
```

Comment:
🚫 **[companylint]** <[custom-template-existence](https://github.com/datenanfragen/data/issues/1120)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record specifies `custom-access-template` of `access-evangelische-kirche` but that isn't available for `request-language` of `it`.